README.md

```diff
@@ -1 +1,12 @@
-# Hello-world
```

Comment:
Please add a professional README file. A README should have the steps to run the project locally. You may use [this](https://github.com/microverseinc/readme-template) tempelate from Microverse.